widgets/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/RecurrenceEditor.java

```diff
@@ -1457,6 +1464,30 @@ public void setOnChangeHandler( ICallback<IChangeHandler> handler ) {
     this.onChangeHandler = handler;
   }
 
+  public boolean getEnableSafeMode() {
+    return detailsPanel.getEnableSafeMode();
+  }
+
+  public void setEnableSafeMode( boolean enableSafeMode ) {
+    detailsPanel.setEnableSafeMode( enableSafeMode );
+  }
+
+  public boolean getGatherMetrics() {
+    return detailsPanel.getGatherMetrics();
+  }
+
+  public void setGatherMetrics( boolean gatherMetrics ) {
+    detailsPanel.setGatherMetrics( gatherMetrics );
+  }
+
+  public String getLogLevelLb() {
```

Comment:
Why is `Lb`, presumably a suffix for "Listbox", relevant in this property name? Seems like a private detail.